sdk/eventgrid/eventgrid/test/cloudEventDistributedTracingEnricherPolicy.spec.ts

```diff
@@ -7,12 +7,12 @@ import {
   PipelineRequest,
   PipelineResponse,
   createPipelineRequest,
-  SendRequest
-} from "@azure/core-https";
+  SendRequest,
```

Comment:
Uh oh, I think you're using a different prettier config!

Reply:
😓

Reply:
I remember this is default of eslint latest version I like it as trailing comma actually makes it easier to modify the list.  One day we will move to the new version...

Reply:
oh wait, it's prettier, ` trailingComma: "es5"` option